pkg/apply/taskrunner/runner.go

```diff
@@ -42,6 +43,7 @@ type Options struct {
 	PollInterval     time.Duration
 	UseCache         bool
 	EmitStatusEvents bool
+	InventoryPolicy  inventory.InventoryPolicy
```

Comment:
Why is the inventory policy required here?

Reply:
Not needed. Removed it.